### ISSUE TITLE
Texture badly position using beginTextureFill on WebGL2

### ISSUE THREAD

**Issue by Makio64** [pixijs/pixijs#8744]
I got into a weird bug while testing webgl2 + pixi, my current background texture is like crop at the wrong position.

### Expected Behavior
WebGL2 act like WebGL1

### Current Behavior
The crop appear  overlay appear on top of half the scene when WebGL2 enabled using the following command : `settings.PREFER_ENV = ENV.WEBGL2`

```
		this.rect = new Graphics()
		this.rect.beginTextureFill({ texture: Assets.get('river/bg-gradient') })
		this.rect.drawRoundedRect(20, 72, width, height, 20)
		this.rect.endFill()
		this.addChild(this.rect)
```

### Environment

- **`pixi.js` version**: 6.5.6
- **Browser & Version**: Chrome latest
- **OS & Version**: _e.g. MacOS latest ( not metal )

<img width="407" alt="Screenshot 2022-10-17 at 17 43 55" src="https://user-images.githubusercontent.com/213351/196222914-cb9f8c41-5e8e-4b61-b333-a9a803d21f0b.png">
<img width="403" alt="Screenshot 2022-10-17 at 17 44 02" src="https://user-images.githubusercontent.com/213351/196222921-88682574-9aab-4690-9fcc-d31b461e04d9.png">

**Comment by bigtimebuddy** [comment]
Thanks for the issue. Could you please provide a minimum reproduction of this issue?

**Comment by Makio64** [comment]
> Thanks for the issue. Could you please provide a minimum reproduction of this issue?

Yes, is there a basic template I can use as a starter for the reproduction example?

**Comment by SuperSodaSea** [comment]
> Yes, is there a basic template I can use as a starter for the reproduction example?

Have a try on https://pixiplayground.com/

**Comment by bigtimebuddy** [comment]
You can also try this: 

https://codesandbox.io/s/pixijs-simple-template-dlwjp9

Codesandbox is much easier for adding assets.

**Comment by Zyie** [comment]
Hi there! 👋

We're doing some housekeeping and closing out older issues that haven't had recent activity.

PixiJS has evolved significantly since this issue was opened. If you're still experiencing this issue on PixiJS v8 (latest), please open a new issue with updated details and a minimal reproduction case.

Thanks for your understanding and for helping us keep the issue tracker focused and maintainable! 🩷
